### ISSUE TITLE
Cannot specify task as complete

### ISSUE THREAD

**Issue by alinademi** [kiblee/tod0#40]
**Describe the bug**

First of all this is an amazing concept and cool cli tool. 👏🏼 

### When marking task as complete I get the following error:

```console
/opt/homebrew/lib/python3.9/site-packages/todocli/utils/datetime_util.py:163: PytzUsageWarning: The localize method is no longer necessary, as this time zone supports the fold attribute (PEP 495). For more details on migrating to a PEP 495-compliant implementation, see https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html
  local_dt = tz.localize(dt, is_dst=None)
```

The error shows with `todocli` or `tod0` regardless.

**the task is marked as complete** but the error pops up.

**Comment by kiblee** [comment]
Hi @alinademi . Fixed this in the recent update. Please upgrade `tod0` with `pip install --upgrade tod0`. Thanks!

**Comment by kiblee** [comment]
Fixed in #39
